vault/logical_system.go

```diff
@@ -2435,14 +2435,21 @@ func (b *SystemBackend) handlePoliciesSet(policyType PolicyType) framework.Opera
 			return nil, err
 		}
 
+		name := data.Get("name").(string)
```

Comment:
I believe there is a wrapper `GetString` that does the safe string casting.

Reply:
Good thing for me to know about! I just looked at it though and it seems like the method (I assume you mean the one from go-secure-stdlib/strutil) takes a map[string]interface{}, whereas data is actually a *framework.FieldData.

Reply:
```suggestion
		name := req.GetString("name")
```

I believe it's this one.

Reply:
As discussed offline, we're going to keep this one as is since this is a line from the original code, which seems to be intentionally passing in and using data *framework.FieldData instead of the string from the logical.Request.